web-client/views/upload.jade

```diff
@@ -67,20 +64,15 @@ html
                       a(href='#' id='exportAsExcelWkbk') To Excel Workbook
                     //- Need to omit the divider too.
                     li(class='divider' style='display: none')
-                    li(class='startDisabled disabled unweighted export')
+                    li(class='startDisabled disabled')
                       a(href='#' id='exportAsUnweightedSif') To Unweighted SIF
-                    li(class='startDisabled disabled weighted export')
+                    li(class='startDisabled disabled')
                       a(href='#' id='exportAsWeightedSif') To Weighted SIF
                     li(class='divider')
-                    li(class='startDisabled disabled unweighted export')
+                    li(class='startDisabled disabled')
                       a(href='#' id='exportAsUnweightedGraphMl') To Unweighted GraphML
-                    li(class='startDisabled disabled weighted export')
+                    li(class='startDisabled disabled')
```

Comment:
These classes can actually be useful in the future (maybe even now) for easily picking out the commands that have to do with export or with weightedness or both.